bucket/dukto.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "6",
+    "description": "An easy file transfer tool designed for LAN use.",
+    "homepage": "https://www.msec.it/blog/dukto/",
+    "license": "GPL-2.0-only",
```

Comment:
The license is definitely not GPL

![A](https://i.imgur.com/ebbYOiG.png)

```suggestion
    "license": "Freeware",
```

Reply:
> The license is definitely not GPL

I'd suggest that it is. (Maybe multi-license?)

![image](https://user-images.githubusercontent.com/24728204/100376941-47f45a80-3008-11eb-89ba-c51a0cd6d965.png)

Reply:
Also on the homepage.

![image](https://user-images.githubusercontent.com/24728204/100378138-72471780-300a-11eb-9473-ae9968aa985e.png)

Reply:
Source code license is irrelevant when there is final product license

Reply:
Weird. Getting confused already.